pkg/cmd/admin/policy/modify_scc.go

```diff
@@ -0,0 +1,234 @@
+package policy
+
+import (
+	"errors"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kclient "k8s.io/kubernetes/pkg/client/unversioned"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/util"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	uservalidation "github.com/openshift/origin/pkg/user/api/validation"
+)
+
+const (
+	AddSCCToGroupRecommendedName      = "add-scc-to-group"
```

Comment:
"user U has role R" sounds right
"user U has SCC S" sounds weird, but maybe I just need to repeat it to myself a few more times